### DISCUSSION TITLE
Please Enable Github Sponsorship

### DISCUSSION THREAD

**Discussion by guyewhite** [mido/mido#475] [General]
Would love to send you some funds as a thank you! 

Would you consider enabling Github Sponsors for your account? [https://github.com/sponsors](https://github.com/sponsors)

**Comment by rdoursenaud** [comment]
Thanks for the suggestion.
This is something we need to discuss with @olemb and @bast.
I'm not sure how we could distribute the eventual resources.
Looking forward to reading your thoughts on the matter.

**Reply by rdoursenaud** [reply]
In the meantime, I have enabled sponsors on my personal account if you'd like to support my maintenance efforts: https://github.com/sponsors/rdoursenaud

**Reply by bast** [reply]
I am sorry for late follow-up here. I support this and the resources should be distributed between OMB and RD. My contributions have been minimal and risk staying minimal.
